common/dataSort.js

```diff
@@ -0,0 +1,90 @@
+const R = require('ramda')
+
+const keys = {
+  order: {
+    asc: 'asc',
+    desc: 'desc',
+  },
+}
+
+const serialize = sortCriteria =>
+  R.pipe(
+    R.map(c => `${c.variable} ${c.order}`),
+    R.join(',')
+  )(sortCriteria)
+
+const getHttpParam = sortCriteria =>
```

Comment:
since you are using this method twice with ```JSON.stringify(DataSort.getHttpParam(sort))``` would it make sense to call this method ```DataSort.toHttpParams``` and returns already the stringified version of it?

Reply:
Ok, done